<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSilder_QProgressBar</class>
 <widget class="QWidget" name="QSilder_QProgressBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSilder_QProgressBar</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>481</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="SetProgressBar_2">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="title">
       <string>滑动条设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="ScrollBarlabel">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ScrollBar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="horizontalScrollBar">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="Sliderlabel">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Slider</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="ProgresslBarlabel">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ProgresslBar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="SetProgressBar">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="title">
       <string>进度条设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="textVisible">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>textVisible</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="InvertedAppearance">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>InvertedAppearance</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="recentvalue">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>显示格式-当前值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="percent">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>显示格式-百分比</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="Sure">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Sure</sender>
   <signal>clicked()</signal>
   <receiver>QSilder_QProgressBar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
